metals/src/main/scala/scala/meta/internal/rename/RenameProvider.scala

```diff
@@ -279,21 +295,21 @@ final class RenameProvider(
     if (colonNotAllowed) {
       client.showMessage(forbiddenColonRename(name, newName))
     }
-    symbolIsLocal(symbol) && (!desc.isMethod || (!colonNotAllowed && !isForbidden))
+    symbolIsLocal(symbol, definitionPath) && (!desc.isMethod || (!colonNotAllowed && !isForbidden))
   }
 
-  private def symbolIsLocal(symbol: String): Boolean = {
+  private def symbolIsLocal(
```

Comment:
maybe we could rename it to `isLocalSymbol` ?

also inside it is `symbol.startsWith("local")` but we can simply reuse `symbol.isLocal` which is performing exactly this check.

Reply:
fixed